pkg/server/version.go

```diff
@@ -17,32 +17,29 @@ limitations under the License.
 package server
 
 import (
+	"fmt"
+
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
-
-	"github.com/containerd/cri-containerd/pkg/version"
 )
 
 const (
-	// For now, containerd and runc are bundled with cri-containerd, cri-containerd
-	// version is more important to us.
-	// TODO(random-liu): Figure out how to package cri-containerd and containerd,
-	// and how to version it. We still prefer calling the container runtime "containerd",
-	// but we care both the cri-containerd version and containerd version.
-	containerName        = "cri-containerd"
-	containerdAPIVersion = "0.0.0"
+	containerName = "containerd"
```

Comment:
We should talk about this...
Will our containerd/cri plugin not have it's own version tag? 
Can/Should we change this before we've killed the cri-containerd standalone mode?

I'm not against this direction just dotting i's and crossing t's to make sure one version tag is right answer esp. given that we will be "vendored" in. My gut says while we are vendored we need a version tag and while we have a version tag we probably should report it.

Reply:
Should we report one version in make version and another in this Version() api?
https://github.com/containerd/cri-containerd/blob/master/Makefile#L63